parlai/tasks/natural_questions/build.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import os
+from tqdm import tqdm
+
+import parlai.core.build_data as build_data
+import parlai.utils.logging as logging
+
+DATASET_NAME_LOCAL = 'NaturalQuestions'
+
+
+def _download_with_gsutil(dpath):
```

Comment:
so were we not able to resolve using gsutil with Python?

Reply:
sorry to keep bringing it back here - could we not use google cloud storage python api? e.g. as in this article: https://medium.com/google-cloud/using-google-cloud-storage-5b9d3f570945 (please let me know if you've already explored this!)

Reply:
No problem. I agree that using the gsutil as it is now is not perfect. I started with using the Python client, but ran into some issue with setting up server credentials for download. I will try again to see if there is a way around that.